documentation/userguide/docs/metadata_forms.md

```diff
@@ -0,0 +1,104 @@
+# **Metadata Forms**
+
+**Introduction**
+
+Metadata forms allow users to add structured contextual information to various entities in the data.all platform. By creating and attaching metadata forms, user can standardize and enrich metadata in a customizable way.
+
+Metadata forms serve several key purposes:
+
+- Improve data discovery by enabling more consistent, complete, and meaningful metadata
+- Capture domain-specific or organizational metadata standards
+- Streamline metadata management workflows
+- Search among all entities in data.all based on attached metadata
+
+**Metadata Form lifecycle and usage**
+
+1. User given the permission from data.all administrators can create metadata forms with the Global visibility and visibility for their teams. Owners and Admins of Organizations and Environments can create metadata forms with Environment-Wide and Organization-wide visibility for owned entities.
+2. Once form is created, the owners can add enforcement rules (see the section below).
+3. All changes in metadata form can be performed only by its owner. 
+4. Metadata form can be attached to an entity by the user with sufficient permissions. Permissions are given by the owner or admin of the entity.
+5. Attached metadata forms can be edited or deleted by any user with sufficient permissions.
```

Comment:
What happens then when the metadata form is edited or deleted and it is attached to entities?